serving/ingress/go.mod

```diff
@@ -4,10 +4,12 @@ go 1.14
 
 require (
 	github.com/google/go-cmp v0.5.2
-	github.com/openshift/api v0.0.0-20200901182017-7ac89ba6b971
+	github.com/openshift/api v0.0.0-20200917102736-0a191b5b9bb0
```

Comment:
hard to guess that's for 4.5 :-)

Reply:
The script is the source of truth here :sweat_smile:. These repos don't cut tags and only cut release branches sadly.